packages/kbn-ui-shared-deps/webpack.config.js

```diff
@@ -7,20 +7,24 @@
  */
 
 const Path = require('path');
-
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
-const TerserPlugin = require('terser-webpack-plugin');
 
-const CompressionPlugin = require('compression-webpack-plugin');
 const { REPO_ROOT } = require('@kbn/utils');
 const { RawSource } = require('webpack-sources');
 
 const UiSharedDeps = require('./src/index');
 
 const MOMENT_SRC = require.resolve('moment/min/moment-with-locales.js');
+const WEBPACK_SRC = require.resolve('webpack');
 
 module.exports = {
+  node: {
+    child_process: 'empty',
+    fs: 'empty',
+  },
+  externals: {
+    module: 'module',
+  },
   mode: 'production',
```

Comment:
Do we know anyone that can check the UI developer tools portion of this?  I thought this would have to be development to get them working.

Reply:
we are using production setup but not minimizing here. Additionally I've added https://github.com/elastic/kibana/pull/105971/files#diff-34ca7c938c7fb012bad5163be6b4952f48a3a1930e91200cc8710113bf7200ddR112 so everything should be working as expected. @shahzad31 could you verify the react profiling when using that branch? Please make sure you run `yarn kbn bootstrap`

Reply:
@jbudz, @shahzad31 is on leaving until September. I've just tested this and the react profiling is now working you can also test it with the react developer tools

Reply:
This fixes the minification issues for me. Component names are now readable again in the dev tools:

<img width="319" alt="Screenshot 2021-07-21 at 09 25 39" src="https://user-images.githubusercontent.com/190132/126456808-a31092e6-c90b-4bac-8464-dad43ab5475c.png">

Thanks a lot for this @mistic - Makes front end development a lot easier! 🥳